common/scripts/install-run.js

```diff
@@ -373,7 +373,12 @@ function installAndRun(packageName, packageVersion, packageBinName, packageBinAr
         cwd: process.cwd(),
         env: process.env
     });
-    return result.status;
+    if (result.status !== null) {
+        return result.status;
+    }
+    else {
```

Comment:
Completely stylistic, but it looks like we use `} else {` in most place in this repo.

Reply:
This is compiled code. That's what the TS compiler produced.

Reply:
Lol. I knew that... 😳